src/components/domains/conversation/Toolbar.js

```diff
@@ -5,33 +5,29 @@ import classnames from "classnames";
 
 export default function Toolbar({
   project,
-  activity,
+  conversation,
   onOpen,
   onExpand,
   canExpand,
   expanded,
-  bookmark,
-  minimal = false,
 }) {
   const expandIcon = expanded ? "chevron-up" : "chevron-down";
   const buttonClasses = "hover:text-tertiary";
   return (
     <div className="group-hover/menu:flex bg-opacity-80 hidden absolute bottom-3 right-6 py-[0.25rem] px-5 space-x-4 bg-gray-200 border-gray-300 rounded-full border dark:bg-gray-900 dark:border-gray-700">
       <BookmarkAction
         project={project}
-        activity={activity}
+        conversation={conversation}
         className={classnames(buttonClasses, "text-sm")}
       />
       {canExpand && (
         <button onClick={onExpand} className={buttonClasses}>
           <FontAwesomeIcon icon={expandIcon} />
         </button>
       )}
-      {!minimal && (
```

Comment:
Just FYI, it was used at some point in the `FunctionOutput` component to keep things tight